incoming-webhook/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+// Response is of type APIGatewayProxyResponse since we're leveraging the
+// AWS Lambda Proxy Request functionality (default behavior)
+type Response events.APIGatewayProxyResponse
```

Comment:
Je suis pas sûr de comprendre ce que fait cette ligne type, d'autant que je ne vois pas de réutilisation de Response ailleurs 🤔

Reply:
You can replace `events.APIGatewayProxyResponse` by your `Response` @gjgd

Reply:
You're right, it's from the boilerplate example and is not used here.

I'm surprised the lint didn't catch it. It catches unused variables but apparently not unused types.

Reply:
Thanks @mourtisma! I think to be consistent with `request events.APIGatewayProxyRequest` we have to either 
1) define a Request type
2 ) remove the Response type. 

I'm ok with either

Reply:
I'm not sure it's worth replacing by Response

Reply:
I agree. You can remove the `Response` alias

Reply:
Removed in https://github.com/gjgd/dont-slack-evil/pull/13/commits/a4027f3d4f645fecbf3a0b3e7c958460c9776e18